extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+                    0, 0, 5
+                ]
+            },
+            "tilingScheme": "quadtree",
+            "tileExtension": "glb",
+            "tilesetExtension": "json",
+            "content": {
+                "levelOffset": 8,
+                "levelOffsetFill": 1
+            },
+            "metadata": {
+                "levelOffset": 8,
+                "levelOffsetFill": 1
+            },
+            "subdivision": {
+                "completeLevels": 7
+            }
+        }
+    }
+}
+```
+
+#### Sparse Octree with Content
+
+##### Base Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricError": 1000,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "cell": [
+                    -90, 30,
+                    0, 30,
+                    90, 30,
+                    180, 30,
+                    0,
+                    10000
+                ]
+            },
+            "tilingScheme": "octree",
+            "tileExtension": "glb",
+            "tilesetExtension": "json",
+            "subdivision": {
+                "bufferView": 0
+            },
+            "content": {
+                "bufferView": 1
+            }
+        }
+    }
+}
+```
+
+#### Quadtree with External Octree
+
+##### Base Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricError": 1000,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    0, 0, 0,
+                    4, 0, 0,
+                    0, 4, 0,
+                    0, 0, 4
+                ]
+            },
+            "tilingScheme": "octree",
+            "tileExtension": "glb",
+            "subdivision": {
+                "completeLevels": 3
+            },
+            "content": {
+                "levelOffset": 4,
+                "levelOffsetFill": 1
+            }
+        }
+    }
+}
+```
+
+##### Root Tile 0 - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricError": 1000,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    0, 0, 0,
+                    4, 0, 0,
+                    0, 4, 0,
+                    0, 0, 4
+                ]
+            },
+            "tilingScheme": "octree",
+            "tileExtension": "glb",
+            "subdivision": {
+                "completeLevels": 3
+            },
+            "content": {
+                "levelOffset": 4,
+                "levelOffsetFill": 1
+            }
+        }
+    }
+}
+```
+
+##### Directory Structure
+
+*Note: Directory structure is not complete. Only for demonstration purposes.*
+ 
+```
+.
+├── tileset.json
+└── R0/
+    ├── tileset.json
+    └── 0/
+        ├── tile.glb
+        ├── 0/
```

Comment:
Something doesn't seem right about this... all `.glb` tiles should be four layers deep `R0/level/x/y.glb`.

Reply:
Updated all examples to have directory structure.